site/layouts/shortcodes/example.html

```diff
@@ -4,17 +4,20 @@
   `args` are all optional and can be one of the following:
     * id: the `div`'s id - default: ""
     * class: any extra class(es) to be added to the `div` - default: ""
-    * js_snippet: add extra JS snippet to StackBlitz - default: `false`
-    * show_preview: if the preview should be output in the HTML - default: `true`
+    * lang: language used to display the code - default: "html"
+    * sb_hide: hide edit StackBlitz button - default `false`
+    * sb_js_snippet: add extra JS snippet to StackBlitz - default: `false`
```

Comment:
Maybe we invert the logic behind these new parameter names to make them easier to parse. Thinking about the following:

- `stackblitz=“false”` (defaults to `true`)
- `add_js=“true”` (defaults to `false`)

Thoughts?

Reply:
Oh yes `stackblitz` is way better than `sb_hide` (with inverted default values).
For more consistency I've renamed `sb_js_snippet` to `stackblitz_add_js` (base on your proposal). IMHO it is important to keep the `stackblitz` prefix; I believe it is more obvious when you use it in a Markdown page, you don't have to ask yourself where this JS is added. But I don't have a strong opinion on this, I still can change it to `add_js` if preferred.
_Updated via https://github.com/twbs/bootstrap/pull/36449/commits/b63f6e3536f8a3f65f1506744901ec5c59c879c3_